src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/arm/deconv_few_edges.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/node_builders/constant.hpp"
+#include "common_test_utils/node_builders/eltwise.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "common_test_utils/node_builders/convolution_backprop_data.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+
+// Subgraph:
+/*
+┌──────────────────┐           ┌──────────────────┐
+│      INPUT       │           │      WEIGHTS     │
+└─────────┬────────┘           └─────────┬────────┘
+          │      ┌──────────────────┐    │
+          └──────┤  DECONVOLUTION   ├────┘
+                 └──┬───────────┬───┘
+                    │           │
+    ┌───────────────┴──┐     ┌──┴───────────────┐
+    │   SQUARED_DIFF   │     │       POWER      │
+    └──────────────────┘     └──────────────────┘
+
+We test work few edges in single port for deconvolution node
```

Comment:
"Verify deconvolution node correctly handles multiple output edges on a single output port"
Also please rename a file to something like deconv_multiple_output_edges.

Reply:
@EgorDuplensky corrected